sdk/servicebus/tests.yml

```diff
@@ -3,7 +3,7 @@ trigger: none
 jobs:
   - template: ../../eng/pipelines/templates/jobs/archetype-sdk-tests.yml
     parameters:
-      PomFilePath: './servicebus/data-plane/pom.xml'
+      PomFilePath: sdk/servicebus/microsoft-azure-servicebus/pom.xml
```

Comment:
We will want to start using a similar structure for the test yml files as we will be for the ci.yml, and pass in the ServiceDirectory.

Reply:
2 issues here: 
1. The path to the pom.xml file is too specific I need to point to the one in `sdk/servicebus`
2. Upgrading all the tests.yml files to use that method today would require that the services live under the `sdk` folder... To keep us from doing piecemeal cleanup work I'll split the pipeline template into `pre-sdk` and the default to which we aspire. That way when the tests.yml files have all been migrated one has only to delete the `pre-sdk` template.